flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/StreamTask.java

```diff
@@ -274,7 +274,7 @@
 
     protected final MailboxProcessor mailboxProcessor;
 
-    final MailboxExecutor mainMailboxExecutor;
+    protected final MailboxExecutor mainMailboxExecutor;
```

Comment:
I can't verify if this is indeed correct, perhaps @fapaul of @AHeise can verify this.